src/torchcodec/_core/custom_ops.cpp

```diff
@@ -381,6 +384,38 @@ OpsAudioFramesOutput get_frames_by_pts_in_range_audio(
   return makeOpsAudioFramesOutput(result);
 }
 
+at::Tensor wrapEncoderPointerToTensor(std::unique_ptr<Encoder> uniqueEncoder) {
+  Encoder* encoder = uniqueEncoder.release();
+
+  auto deleter = [encoder](void*) { delete encoder; };
+  at::Tensor tensor =
+      at::from_blob(encoder, {sizeof(Encoder)}, deleter, {at::kLong});
+  auto encoder_ = static_cast<Encoder*>(tensor.mutable_data_ptr());
+  TORCH_CHECK_EQ(encoder_, encoder) << "Encoder=" << encoder_;
+  return tensor;
+}
+
+Encoder* unwrapTensorToGetEncoder(at::Tensor& tensor) {
+  TORCH_INTERNAL_ASSERT(tensor.is_contiguous());
+  void* buffer = tensor.mutable_data_ptr();
+  Encoder* encoder = static_cast<Encoder*>(buffer);
+  return encoder;
+}
+
+at::Tensor create_encoder(
+    const at::Tensor wf,
+    int64_t sample_rate,
+    std::string_view file_name) {
+  std::unique_ptr<Encoder> uniqueEncoder =
+      std::make_unique<Encoder>(wf, static_cast<int>(sample_rate), file_name);
+  return wrapEncoderPointerToTensor(std::move(uniqueEncoder));
+}
+
+void encode(at::Tensor& encoder) {
+  auto encoder_ = unwrapTensorToGetEncoder(encoder);
+  encoder_->encode();
+}
+
```

Comment:
Let's make these names `create_audio_encoder()` and `encode_audio()` here and in the Python layer as well. Same as the underlying C++ class, if we can make things generic for video later, we can consolidate. But I anticipate we'll probably need different entry points due to different kind of input parameters.

Reply:
OK for `encode_audio()` op to be `encode_audio()`, but unless you feel strongly about I'll leave the `encode()` C++ method as `encode()`, since its class will be changed to `AudioEncoder`.